test/acceptance/features/steps/builder_image.py

```diff
@@ -0,0 +1,52 @@
+from command import Command
+import re
+from openshift import Openshift
+
+
+class BuilderImage(object):
+
+    tag = "19.1.1"
+    namespace = "openshift"
+    image = ""
+
+    openshift = Openshift()
+
+    def __init__(self, image="ubi-quarkus-native-s2i"):
+        self.cmd = Command()
+        self.image = image
+
+    def is_present(self):
+        get_name_output = self.get_name()
+        if (get_name_output is not None):
+            pattern = f'.*{self.image}:{self.tag}'
+            if re.search(pattern, get_name_output) is not None:
+                return True
+        return False
+
+    def get_name(self):
+        output, exit_code = self.cmd.run(
+            f'oc get is {self.image} -n {self.namespace} -o json | jq -rc \'.spec.tags[] | select(.annotations.tags == "builder").from.name\'')
+        if self.image in output:
+            if re.search("not found", output):
+                return None
+            else:
+                return output.rstrip('\n')
+        return None
+
+    def import_and_patch(self):
+        cmd_import = f"oc import-image quay.io/quarkus/{self.image}:{self.tag} -n {self.namespace} --confirm"
+        print(f'===> Import CMD: {cmd_import}')
+        (import_output, exit_code) = self.cmd.run(cmd_import)
+        if exit_code != 0:
+            return False
+        if re.search(f'.*{self.image}\\simported', import_output) is None:
+            return False
+        spec = '{"spec": {"tags": [{"name" : \"%s\", "annotations": {"tags": "builder"}}]}}' % self.tag
+        cmd_patch = f'oc patch is {self.image} -n {self.namespace} -p \'{spec}\''
+        print(f'===> Patch CMD: {cmd_patch}')
+        (patch_output, exit_code) = self.cmd.run(cmd_patch)
+        if exit_code != 0:
+            return False
```

Comment:
Will it be useful to print `patch_output` in case of failure, this might help to troubleshoot.

Reply:
`        if exit_code != 0:
            print(f"cmd-{cmd_patch} issued result is {patch_output} with the exit status {exit_code}")
            return False`

Let me know if this fine

Reply:
Thanks

Reply:
You're already printing `{cmd_patch}` at L46 so it is not necessary here. Just output would be sufficient.

Reply:
done, thanks`print(f"Result for patching image stream is {patch_output} with the exit code {exit_code}")`

Reply:
I don't think we need any statement to point out the output, I think it will be good if we can trim down the sentence to something like, `print(f"Exit Code: {exit_code}\nResult: {patch_output}")`.

Reply:
I think, it is okay to have some statement as well. we can quickly get into this while debugging.

Reply:
Okay keep it but it should not unnecessarily prolix the log.

Reply:
anyways introducing logging will handle as this might go as part of debug messages

Reply:
Okay, you can resolve this conversation.